modules/runner/src/testFixtures/java/org/apache/ignite/internal/ClusterPerClassIntegrationTest.java

```diff
@@ -168,8 +174,46 @@ void stopCluster() {
 
     /** Drops all visible tables. */
     protected static void dropAllTables() {
-        for (Table t : CLUSTER.aliveNode().tables().tables()) {
-            sql("DROP TABLE " + t.name());
+        Ignite aliveNode = CLUSTER.aliveNode();
+
+        for (Table t : aliveNode.tables().tables()) {
+            IgniteImpl ignite = unwrapIgniteImpl(aliveNode);
+            CatalogManager catalogManager = ignite.catalogManager();
+
+            int latestCatalogVersion = catalogManager.latestCatalogVersion();
+            Catalog latestCatalog = catalogManager.catalog(latestCatalogVersion);
+            assert latestCatalog != null;
+
+            TableImpl tableImpl = unwrapTableImpl(t);
+            int tableId = tableImpl.tableId();
+
+            for (CatalogSchemaDescriptor schema : latestCatalog.schemas()) {
+                for (CatalogTableDescriptor table : schema.tables()) {
+                    if (table.id() != tableId) {
+                        continue;
+                    }
+                    sql("DROP TABLE " + QualifiedName.of(schema.name(), table.name()).toCanonicalForm());
+                }
+            }
+        }
+    }
+
+    /** Drops all non-system schemas. */
+    protected static void dropAllSchemas() {
+        Ignite aliveNode = CLUSTER.aliveNode();
+        IgniteImpl ignite = unwrapIgniteImpl(aliveNode);
+        CatalogManager catalogManager = ignite.catalogManager();
+
+        int latestCatalogVersion = catalogManager.latestCatalogVersion();
+        Catalog latestCatalog = catalogManager.catalog(latestCatalogVersion);
+        assert latestCatalog != null;
+
+        for (CatalogSchemaDescriptor schema : latestCatalog.schemas()) {
+            String schemaName = schema.name();
```

Comment:
Schema name should be quoted.
```suggestion
            String schemaName = IgniteNameUtils.quote(schema.name());
```

Reply:
Thanks fixed.